src/verifySignature.js

```diff
@@ -1,20 +1,20 @@
 const crypto = require('crypto');
 const timingSafeCompare = require('tsscmp');
 
-const isVerified = (req) => { 
-  const signature = req.headers['x-slack-signature'];
-  const timestamp = req.headers['x-slack-request-timestamp'];
-  const hmac = crypto.createHmac('sha256', process.env.SLACK_SIGNING_SECRET);
-  const [version, hash] = signature.split('=');
+const isVerified = (req) => {
+    const signature = req.headers['x-slack-signature'];
+    const timestamp = req.headers['x-slack-request-timestamp'];
+    const hmac = crypto.createHmac('sha256', process.env.SLACK_SIGNING_SECRET);
+    const [version, hash] = signature.split('=');
 
-  // Check if the timestamp is too old
-  const fiveMinutesAgo = ~~(Date.now() / 1000) - (60 * 5);
-  if (timestamp < fiveMinutesAgo) return false;
+    // Check if the timestamp is too old
+    const rightNow = Math.floor((Date.now() / 1000));
+    if (Math.abs(rightNow - timestamp) > 300) return false;
```

Comment:
Is there a reason you changed the timeframe here?

Reply:
Frankly _paranoia_ more than anything 😬 I started thinking, what if someone always sent timestamps 5 minutes into the future. It'd always pass this test. 
Seems like Canva also worries about this: https://docs.developer.canva.com/apps/releasing-your-app/verify-request/verify-timestamp
Although we don't: https://github.com/slackapi/bolt-js/blob/a168252f5704117521c3c7c26d1d5a3b59bf491e/src/ExpressReceiver.ts#L319

More than happy to revert it if you prefer.

Reply:
No that seems fair. 

At some point we'll convert this sample to bolt anyway, so it'll be moot.